internal/parser/scanner/ruleset/ruleset_default.go

```diff
@@ -23,13 +24,17 @@ var (
 	// defaultLinefeedDetector is the linefeed detector that this ruleset allows
 	defaultLinefeedDetector   = matcher.RuneWithDesc("linefeed", '\n')
 	defaultStatementSeparator = matcher.RuneWithDesc("statement separator", ';')
+	defaultDecimalPoint       = matcher.RuneWithDesc("decimalPoint", '.')
```

Comment:
don't use camel case when the line before uses a space
```suggestion
	defaultDecimalPoint       = matcher.RuneWithDesc("decimal point", '.')
```

Reply:
Ah cool